cmd/server/grpc/main.go

```diff
@@ -110,15 +118,19 @@ func main() {
 		os.Exit(-1)
 	}
 
+	log.Init(conf.Log.Level)
+
 	log.Infof("--- Starting SFU Node ---")
 	lis, err := net.Listen("tcp", addr)
 	if err != nil {
 		log.Panicf("failed to listen: %v", err)
 	}
 	log.Infof("SFU Listening at %s", addr)
 	s := grpc.NewServer()
-	pb.RegisterSFUServer(s, &server{
-		sfu: sfu.NewSFU(conf.Config),
+	sfuServer := NewSFUServer(conf.Config)
+	pb.RegisterSFUService(s, &pb.SFUService{
```

Comment:
I feel like we didn't need to change this?

I think it can be like before:
```
pb.RegisterSFUServer(s, &server{
    sfu: sfu.NewSFU(conf.Config),
})
```

and revert changes above too (other than line 121)

Reply:
my compile is version 7, i guess, it changed things